waiter/src/waiter/core.clj

```diff
@@ -571,9 +571,10 @@
                                   [:state http-client]
                                   make-basic-auth-fn service-id->password-fn]
                            (fn make-http-request-fn [service-id location auth-user-map request-method headers body]
-                             (let [service-password (service-id->password-fn service-id)]
-                               (pr/make-http-request http-client make-basic-auth-fn request-method location headers body
-                                                     service-password auth-user-map initial-socket-timeout-ms output-buffer-size))))
+                             (let [service-password (service-id->password-fn service-id)
+                                   request {:request-method request-method :headers headers :body body}]
```

Comment:
This function's API doesn't support cookies at all, I just needed to update it because I added the parameter to `make-http-request`